sdk/nodejs/awsMixins.ts

```diff
@@ -39,6 +39,23 @@ import * as pulumiAws from ".";
 // that ends up returning `require("aws-sdk")` inside of it.
 
 declare module "." {
+    /**
+     * Provides quick access to the "aws-sdk" SDK (https://github.com/aws/aws-sdk-js) within a
+     * javascript callback function that is serialized into an AWS Lambda.  Inside a Pulumi app
+     * a program can be written like so:
+     *
+     * ```ts
+     *    import * as aws from "@pulumi/aws";
+     *
+     *    // ...
+     *
+     *    // inside the callback function for an AWS Lambda:
+     *    const client = new aws.runtime.DynamoDB.DocumentClient(...opt-args...);
+     * ```
+     *
+     * Note: this property will give you the aws-sdk module that AWS automatically includes
+     * with any javascript Lambda.
+     */
     export const runtime: typeof import("aws-sdk");
```

Comment:
i definitely thought about that.  But found it a bit strange to use the term 'sdk' inside an 'sdk' :)  But if you do like that better, i think i'm totally ok with it.    It also doesn't really need to be called 'runtime' since it is totally usable at deployment time as well.